Cargo.toml

```
@@ -24,8 +25,8 @@ all-algorithms = ["brotli", "bzip2", "deflate", "gzip", "lzma", "xz", "zlib", "z
 # algorithms
 deflate = ["flate2"]
 gzip = ["flate2"]
-lzma = ["xz2"]
-xz = ["xz2"]
+lzma = ["liblzma"]
+xz = ["liblzma"]
```

```suggestion
xz = ["lzma"]
xz2 = ["xz"]
```

Would also need xz2 to maintain backwards compatibility since it is implicitly defined